crates/control/src/motion/step_planner.rs

```diff
@@ -22,16 +23,9 @@ pub struct CreationContext {}
 
 #[context]
 pub struct CycleContext {
-    motion_command: Input<MotionCommand, "motion_command">,
+    parameters: Parameter<StepPlannerParameters, "step_planner">,
```

Comment:
why did you reintroduce this struct? If I read the changes correctly, you only need the max step size in the walking engine. Additionally, we want to remove the `types` crate over time. So if we really want to have such a struct, we should find a different place for it.

Furthermore, the `StepPlannerParameters` struct seemed to be still `types` before your changes, previously unused...